### PR TITLE
Limit command output to fix problem with crashed browser tab

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
With help of @vitaliy-guliy  Limit command output to fix problem with crashed browser tab

### What issues does this PR fix or reference?
Fast fix discussed here https://github.com/eclipse/che/issues/7008#issuecomment-339650502
Now we can build che in che

#### Release Notes
Command output limited to 500 lines

### CODE REVIEW

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/console/OutputConsoleViewImpl.java [hunk 1]
```diff
@@ -317,6 +317,11 @@ public void print(String text, boolean carriageReturn) {
 
   @Override
   public void print(final String text, boolean carriageReturn, String color) {
+
+    if (consoleLines.getElement().getChildCount() > 500) {
```
Comment (tolusha) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/console/OutputConsoleViewImpl.java:321]: It is better to have a constant but not a magic number ^)

Comment (skabashnyuk) [reply ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/console/OutputConsoleViewImpl.java:321]: I would say this change would not make this fast and dirty fix less dirty.

**Comment by vitaliy-guliy** [comment]
Looks good for start

**Review by vitaliy-guliy — APPROVED** [review]

**Review by vparfonov — APPROVED** [review]

**Review by benoitf — DISMISSED** [review]
it needs to be parametrized (or use buffering)
we shouldn't do "quick and dirty" code

**Comment by benoitf** [comment]
note that you can use `quiet`flag as well for maven

**Comment by skabashnyuk** [comment]
What kind of parametrization? 
@benoitf are you ready to implement that until next Thursday? because https://github.com/eclipse/che/issues/7008 is a real blocker.
FYI @AndrienkoAleksandr is going to make some test here https://github.com/eclipse/che/issues/7029 but it would not be very quick

**Comment by benoitf** [comment]
parametrization is to not hardcode limits in a product

**Comment by gazarenkov** [comment]
makes no sense to improve workaround,
might be the choice if we will not be able to make something better after @AndrienkoAleksandr investigation

**Pull request merged**